runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -504,14 +504,8 @@ public int getSkillExperience(Skill skill)
 
 		if (skill == Skill.OVERALL)
 		{
-			int totalExperience = 0;
-
-			for (int experience : experiences)
-			{
-				totalExperience += experience;
-			}
-
-			return totalExperience;
+			logger.debug("getSkillExperience called for {}!", skill);
+			return (int) getOverallExperience();
```

Comment:
Why not just change return value to long? This basically intentionally returns wrong value, and also then you need to do stuff like you do in unpause method

Reply:
I thought about this but it is used a lot of places, most of which aren't getting overall xp.